client/src/components/Home/index.jsx

```diff
@@ -12,6 +12,13 @@ class Home extends Component {
         this.handleSubmit = this.handleSubmit.bind(this)
     }
 
+    componentWillReceiveProps(nextProps) {
+        if(nextProps.currLocation.location) {
+            const {latitude, longitude} = nextProps.currLocation.location
+            this.props.history.push(`/search?latitude=${latitude}&longitude=${longitude}`)
```

Comment:
In our db, there is a convention that we store longitude first and then latitude, so to avoid any kind of confusions, let's keep longitude also before latitude if that sounds good?

Reply:
Sounds good